### DISCUSSION TITLE
Is it possible to set backup dir through config?

### DISCUSSION THREAD

**Discussion by djotanov** [apache/kvrocks#475] [Q&A] [answered]
I was looking through the config file but I don't see a way to specify backup dir independently of data dir. Is there some undocumented way maybe?

**Comment by git-hulk** [comment, marked as answer]
Yes, we can use the `backup-dir` to specify the backup dir but didn't describe at the config file. Would also show it in next release. Thanks for your feedback.

**Reply by djotanov** [reply]
Thanks, thats helpful.
What about restore from backup? How is that performed?

**Reply by git-hulk** [reply]
RocksDB offers the `ldb` tool to restore the backup to the DB dir but needs to install the RocksDB tool first. This is the shell which wraps the ldb: https://github.com/facebook/rocksdb/blob/main/tools/restore_db.sh

**Reply by djotanov** [reply]
Thanks. Maybe, as a suggestion for future versions, we could have a new command for restore from latest backup? 
When using docker (kubernetes deployment scenarios), it becomes trickier to automate database restore if we need to use external tool.
